malaria24/ona/models.py

```diff
@@ -63,11 +63,17 @@ def alert_new_case(sender, instance, created, **kwargs):
             instance.facility_code,))
 
     for ehp in ehps:
-        if ehp.phone_number:
+        if ehp.phone_number and ehp.email_address:
```

Comment:
Can we make this generic please.
In the models we are working with, there are EHPs, district managers, case workers and nurses.

If we call it `ehp` we will have to shoe-horn this for other people... making for a "messy" implementation.